test/conformance/ingress/run.go

```diff
@@ -32,6 +32,7 @@ func RunConformance(t *testing.T) {
 	t.Run("grpc", TestGRPC)
 	t.Run("grpc/split", TestGRPCSplit)
 
+	t.Run("headers/probe", TestProbeHeaders)
```

Comment:
I think we need to add this under alpha or beta and socialize the new addition a bit.

Reply:
Alright. I move it to beta as it is not a brand-new feature but we just need to socialize it.